x-pack/test/api_integration/deployment_agnostic/apis/observability/alerting/custom_threshold/avg_us_fired.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
```

Comment:
Not sure why, but this one failed on MKI due to a mismatch in the number of generated documents. So, I am checking if this logic works for `avg_pct_fired` since we use `data_forge` there instead of `synthtrace`.

<img width="1115" alt="Screenshot 2024-12-06 at 18 01 00" src="https://github.com/user-attachments/assets/496d57a0-9f64-463d-9526-57d6bfb37ee2">